packages/host/app/components/operator-mode/profile-info-popover.gts

```diff
@@ -192,43 +208,50 @@ export default class ProfileInfoPopover extends Component<ProfileInfoPopoverSign
     return await this.realmServer.fetchCreditInfo();
   });
 
+  private get isLoading() {
+    return this.fetchCreditInfo.isLoading;
+  }
+
   private get plan() {
     return this.fetchCreditInfo.value?.plan;
   }
 
+  private get planName() {
+    return this.plan?.name;
+  }
+
   private get creditsIncludedInPlanAllowance() {
-    return this.fetchCreditInfo.value?.creditsIncludedInPlanAllowance;
+    return this.fetchCreditInfo.value?.plan?.creditsIncluded;
   }
 
-  private get creditsUsedInPlanAllowance() {
-    return this.fetchCreditInfo.value?.creditsUsedInPlanAllowance;
+  private get creditsAvailableInPlanAllowance() {
+    return this.fetchCreditInfo.value?.creditsAvailableInPlanAllowance;
   }
 
   private get extraCreditsAvailableInBalance() {
     return this.fetchCreditInfo.value?.extraCreditsAvailableInBalance;
   }
 
   private get monthlyCreditText() {
-    return this.creditsUsedInPlanAllowance != undefined &&
-      this.creditsIncludedInPlanAllowance != undefined
-      ? `${
-          this.creditsIncludedInPlanAllowance - this.creditsUsedInPlanAllowance
-        } of ${this.creditsIncludedInPlanAllowance} left`
-      : '';
+    return this.creditsAvailableInPlanAllowance != null &&
+      this.creditsIncludedInPlanAllowance != null
+      ? `${this.creditsAvailableInPlanAllowance} of ${this.creditsIncludedInPlanAllowance} left`
+      : null;
   }
 
   private get isOutOfCredit() {
     return (
       this.isOutOfPlanCreditAllowance &&
-      this.extraCreditsAvailableInBalance == 0
+      (this.extraCreditsAvailableInBalance == null ||
+        this.extraCreditsAvailableInBalance == 0)
     );
   }
 
   private get isOutOfPlanCreditAllowance() {
     return (
-      this.creditsUsedInPlanAllowance &&
-      this.creditsIncludedInPlanAllowance &&
-      this.creditsUsedInPlanAllowance >= this.creditsIncludedInPlanAllowance
+      this.creditsAvailableInPlanAllowance == null ||
```

Comment:
Why checking for null? Isn't `this.creditsAvailableInPlanAllowance <= 0` enough to check whether user has any plan credits left?

Reply:
Actually, it's just to satisfy the type check, since we decided to not throw NotFoundError from endpoint if no subscription found in database, instead we make creditsAvailableInPlanAllowance value can be null.